src-tauri/src/pipeline.rs

```diff
@@ -63,11 +69,15 @@ pub async fn pipeline(
     response_function: String,
     fisheye_correction_cal: String,
     vignetting_correction_cal: String,
+    photometric_adjustment_cal: String,
```

Comment:
The photometric and neutral density calibration file arguments on the backend need to match the arguments on the frontend, except that in Rust the argument names will be snake case and on the front end it will be camel case. So either the FE or BE argument names need to be updated so they match